### DISCUSSION TITLE
Making position not fixed

### DISCUSSION THREAD

**Discussion by ghost** [matteobruni/tsparticles#2829] [Q&A] [answered]
I only need the particles apply on some section, but its position is fixed on the element and I can't even use CSS to overwrite it.

Is there an option to disable it or what can I do to make it not fixed?

**Comment by matteobruni** [comment, marked as answer]
I answered this yesterday, you can find all the details here: https://github.com/matteobruni/tsparticles/issues/1277#issuecomment-1008909977
